test/multiverse/suites/rake/Rakefile

```diff
@@ -45,14 +45,20 @@ namespace :named do
   task :all => [:'named:before', :'named:during', :'named:after']
 
   task :before do
+    a = NewRelic::Agent::Tracer.start_segment(name: 'custom_before')
     puts "named:before"
+    a.finish
   end
 
   task :during do
+    a = NewRelic::Agent::Tracer.start_segment(name: 'custom_during')
     puts "named:during"
+    a.finish
   end
 
   task :after do
+    a = NewRelic::Agent::Tracer.start_segment(name: 'custom_after')
     puts "named:after"
+    a.finish
```

Comment:
I think I'm a bit confused about these changes. Are rake tasks supposed to be automatically instrumented? Does adding `start_segment` here make them manually instrumented?

Reply:
The rake tasks are automatically instrumented yeah, these start segments are just extra segments. 
Originally I had net http calls in these tasks, and I added those so we could make sure that anything happening inside of the task was being appropriately recorded inside of the tasks transaction. I didn't want to keep real network calls though in the tests, so I converted it to create a custom segment instead. The tests are checking to make sure the rake segments are being created also, this is just extra.

Reply:
Awesome. Thanks for clarifying.